packages/openapi-typescript/test/transform/schema-object/array.test.ts

```diff
@@ -179,7 +245,7 @@ describe("transformSchemaObject > array", () => {
       {
         given: {
           type: "array",
-          items: { type: "number" },
+          items: false,
```

Comment:
This specifically is the breaking change I’m referring to—we’ll need to ensure that the original test case of `items: { type: "number" }` stays tested, and it generates the same output. While we’re on version 7.x this can’t change (and we can revisit it when it’s time to release 8.x).

In general, modifying an original test is a good hint that it may be a breaking change.